src/AuditTrailModule/javasource/test_crm/tests/TestAuditInheritance.java

```diff
@@ -45,7 +47,7 @@ public void beforeTesting() throws CoreException {
 	}
 
 	@Test
-	public void testCreatedRecord() throws InterruptedException, CoreException {
+	public void testCreateRecord() throws InterruptedException, CoreException {
```

Comment:
I wonder which API call throws `InterruptedException`. Seems unnecessary to me.

Reply:
Probably some bad copy/pasting from the unit testing module. Removing.